### PR TITLE
Adding my art

### PR DESCRIPTION
**Pull request by AsharAA**
# Zero To Mastery - Canvas 2022 🎉🎉

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
- INFO ABOUT YOUR PULL REQUEST GOES HERE (Please be as descriptive as possible) 🤜

## Breaking changes
- IF ANYTHING YOU'RE COMMITTING WOULD BREAK SOMETHING, INCLUDE HERE WHAT WOULD BREAK
- IF YOU HAVE NO BREAKING CHANGES, ENTER 'None'

## Screenshots
- IF YOU HAVE ANY SCREENSHOTS, INCLUDE THEM HERE.
- IF YOU HAVE NO SCREENSHOTS, ENTER 'None'

### CODE REVIEW

File: src/art.js [hunk 1]
```diff
@@ -49,6 +49,14 @@
     "author_name": "coder@hacktoberfest",
     "author_github_url": "https://github.com/Adwaith123/"
   },
+  {
```
Comment (sophiabrandt) [inline src/art.js:52]: Please rollback any changes to this file, see README

File: src/art/Mwheel/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=h, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Special+Elite&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>THE WHEEL</title>
+</head>
+<body>
+    <h1>MOTION WHEEL</h1>
+  <img src="image/car.png" alt="wheel-image">
```
Comment (sophiabrandt) [inline src/art/Mwheel/index.html:13]: Please provide the license to the image. Images are normally under copyright, so we need to make sure that we don't violate copyright.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

**Comment by sophiabrandt** [comment]
Please fill out the request template.

File: src/art/Mwheel/index.html (hunk 2 above)
Comment (AsharAA) [reply src/art/Mwheel/index.html:13]: It's a copyright free image took from flaticon

Comment (sophiabrandt) [reply src/art/Mwheel/index.html:13]: According to [their website](https://support.flaticon.com/s/article/Is-Flaticon-for-free?language=en_US) you need to add an attribution line.

Please point to the code where you do that. I haven't seen it, but maybe I've overlooked it. Thank you.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please add a screenshot as `src/art/Mwheel/icon.png`. The easiest way is probably to rename the existing `Mwheel.png`.

**Review by MattCSmith — CHANGES_REQUESTED** [review]
Please make the two adjustments to ensure the `Readme.md` & `Art.js` remain untouched. 

Otherwise this looks good to go 👍

File: src/art.js [hunk 3]
```diff
@@ -49,6 +49,8 @@
     "author_name": "coder@hacktoberfest",
     "author_github_url": "https://github.com/Adwaith123/"
   },
+
```
Comment (auxfuse) [inline src/art.js:52]: Can you remove both of these blank rows please?

**Review by auxfuse — CHANGES_REQUESTED** [review]
`art.js` needs changes as blank rows added on line 52 & 53

**Review by auxfuse — APPROVED** [review]
LGTM 🎉

**Pull request merged**